.configure

```diff
@@ -14,8 +14,13 @@
       "encrypt": true
     },
     {
-      "file": "android/WPAndroid/sentry.properties",
-      "destination": "sentry.properties",
+      "file": "android/WPAndroid/wordpress-sentry.properties",
+      "destination": "WordPress/src/wordpress/sentry.properties",
+      "encrypt": true
+    },
+    {
+      "file": "android/WPAndroid/jetpack-sentry.properties",
+      "destination": "WordPress/src/jetpack/sentry.properties",
```

Comment:
After deep investigation inside the plugin code, we finally discovered (even though it's not documented anywhere we could find in the sentry doc…) that the [Sentry gradle plugin tries multiple paths](https://github.com/getsentry/sentry-android-gradle-plugin/blob/main/plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPropertiesFileProvider.kt#L18-L64) when searching for the `sentry.properties` file.

You can see which one it tries to find by running `./gradlew -q -d | grep '\[sentry\]'` and seeing the various `Looking for sentry.properties at: …` logs it tries.

Here I'm taking advantage of this hidden feature in the plugin to provide different `sentry.properties` – and thus different `defaults.project` values – for the `uploadSentryProguardMappings${variant}` gradle tasks (that the sentry plugin registers) to find, depending on the variant associated with each task it declares.